rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -611,6 +612,7 @@ def decode_transaction_hashes(
             )
 
         self._post_process(refresh_balances=refresh_balances)
+        maybe_detect_new_tokens(self.database)
         return events
```

Comment:
this doesn't make any external request right?

Reply:
no it only process the data in the db